pallets/automation-time/src/lib.rs

```diff
@@ -921,7 +928,12 @@ pub mod pallet {
 			amount: BalanceOf<T>,
 			task_id: T::Hash,
 		) -> Weight {
-			match T::NativeTokenExchange::transfer(&sender, &recipient, amount) {
+			match T::Currency::transfer(
+				&sender,
+				&recipient,
+				amount,
+				ExistenceRequirement::KeepAlive,
```

Comment:
Extra parameter because `NativeTokenExchange`'s `transfer` adds this before forwarding to `Currency`'s `transfer`.